src/cart/src/utest/test_gurt.c

```diff
@@ -1121,10 +1136,15 @@ test_gurt_hash_decref(void **state)
 	assert_int_equal(rc, 0);
 	assert_int_equal(entry->tl_ref, 2);
 
-	/* Drop 20 refs, which should fail but remove and free the descriptor */
+	/* Drop 20 refs, which should fail but not
+	 * remove or free the descriptor */
```

Comment:
(style)  Block comments use a trailing */ on a separate line

Reply:
should fix this